modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -580,6 +580,11 @@
 		<td>//div[@class='mt-1']</td>
 		<td></td>
 	</tr>
+	<tr>
+	<td>ERROR_MESSAGE_INPUT_DOES_NOT_MATCH</td>
+	<td>//div[contains(@class,'form-feedback-item')]</td>
+	<td></td>
+	</tr>
```

Comment:
Hello Jhosseph, this path already exists in other places on the portal! You can replace it with _Message#WARNING_FEEDBACK_ :)